src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.module.scss

```diff
@@ -1,4 +1,4 @@
-.attentionBoxLink {
+a.attentionBoxLink {
```

Comment:
is the `a` tag really necessary?

Reply:
yes for specificity (we overriding the default style we implement for link in text component)